metamath.c

```diff
@@ -2959,6 +2973,8 @@ void command(int argc, char *argv[])
       i = 5;  /* # arguments with only / HTML or / ALT_HTML */
       if (noVersioning) i = i + 2;
       if (switchPos("/ TIME")) i = i + 2;
+      /* 7-Jul-2017 added MathML/STS */
+      if (switchPos("/ STS")) i = i + 1;

```

Comment:
@wlammen wrote:

> Suspicious value of i. The other options / TIME or / NO_VERSIONING add 2 to the number of options, seemingly counting the slash and the tag as different entries. Why is this not done for / STS? If the count is correct, IMO a comment should clarify the underlying logic.

Interesting. So, a typical `SHOW STATEMENT` command looks like this: 
``` SHOW STATEMENT syl / HTML ```
That would be g_rawArgs = 5, which already includes 2 arguments for the `/ HTML`.
As you correctly guessed, for `/ NO_VERSIONING` and `/ TIME`, there are two more arguments, therefore the +2.
In the case of STS, the `/ STS` would already be accounted for in the 5 (taking the place of the `/ HTML`), so that's not why the +1 is for. Rather, in the case of STS, there is one more argument, namely the name of the output processing, e.g. `mathml`. That is what the +1 is for.

I'll add a comment to make that clearer.

Reply:
I am subject to a learning curve as well while reviewing, wrt both to your source code and the review style.